protocols.go

```diff
@@ -46,6 +47,7 @@ var Protocols = []Protocol{
 	Protocol{P_IP6, 128, "ip6", CodeToVarint(P_IP6)},
 	// these require varint:
 	Protocol{P_SCTP, 16, "sctp", CodeToVarint(P_SCTP)},
+	Protocol{P_ONION, 10, "onion", CodeToVarint(P_ONION)},
```

Comment:
`10` seems off? shouldn't this be `80` ? (sorry for being inconsistent, the multihash standard is bytes for compactness, this is in bits. arguably should be bytes).